src/test/java/com/google/javascript/clutz/partial/optional_export_object_property.js

```diff
@@ -0,0 +1,17 @@
+goog.module('optional.export.object.property');
+goog.module.declareLegacyNamespace();
```

Comment:
just goog.provide if you really need a provide here. Even better if just a goog.module works for the repro.

Reply:
Unfortunately not.  Neither of those get the right code path through clutz.